### DISCUSSION TITLE
Delay in create method

### DISCUSSION THREAD

**Discussion by yousufhasan** [prisma/prisma#5458] [Q&A]
Hi, While saving v. large nested objects using for.eg prisma.acccounts.create({data}) prisma is start running the queries after a lag of few minutes i.e. i can't see any query on my logs until after a few minutes. The larger the object the larger is the delay? Is it a known issue ?

**Comment by ryands17** [comment]
@yousufhasan 👋 
Could you share your example data that you are passing so that I can reproduce?

**Reply by yousufhasan** [reply]
Well it is a very large object almost 1 million records. so all i am doing is trying to call nested writes using for e.g prisma.accounts.create({data}); where data includes many subentities and children entitities. It doesn't take that longer if the object size is smaller. The problem is that if we pass a large object to .create() method i expect get queries on my terminal immediately but it takes a few minutes and only then it starts  running the queries which makes the whole operation even more slower.

**Reply by ryands17** [reply]
This seems like a performance issue, so I would suggest creating an issue [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=bug_report.md&title=) as we would need to generate a long nested list of items to check with. We would also need the database and its version so that we can reproduce it with the correct versions and find the root cause.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
